websocket.go

```diff
@@ -545,7 +545,10 @@ func (c *wsConn) handleWsConn(ctx context.Context) {
 		var timeoutCh <-chan time.Time
 		if timeoutTimer != nil {
 			if !timeoutTimer.Stop() {
-				<-timeoutTimer.C
+				select {
+				case <-timeoutTimer.C:
+				default:
```

Comment:
adding a default here basically makes the select pointless. Should we just remove the timer altogether?

Reply:
This was doing the right thing (the select isn't needed). The channel needs to be drained when Stop returns false -> https://golang.org/pkg/time/#Timer.Stop

Reply:
Hrm, in local testing it.. apparently is needed
```
goroutine 130 [chan receive, 2 minutes]:
github.com/filecoin-project/go-jsonrpc.(*wsConn).handleWsConn(0xc00026e240, 0x346c880, 0xc000042848)
	/home/magik6k/gohack/github.com/filecoin-project/go-jsonrpc/websocket.go:548 +0x1208
created by github.com/filecoin-project/go-jsonrpc.websocketClient
	/home/magik6k/gohack/github.com/filecoin-project/go-jsonrpc/client.go:262 +0x2cf

```

Reply:
Ah, I see why it makes sense, `Stop()` will tell us that the timer has fired, but we may have already drained the channel below.